.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: java
+jdk: openjdk8
+sudo: required
+dist: trusty
+
+install:
+- wget http://www-us.apache.org/dist/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
+- tar xzf apache-maven-3.3.9-bin.tar.gz
+- curl -s -L https://github.com/mozilla/smarthome/releases/latest | egrep -o /mozilla/smarthome/releases/download/.*/m2repository.tar.gz | wget --base=http://github.com/ -i -
+- tar xf m2repository.tar.gz -C $HOME/.m2/repository/
+
+before_script:
+- rm $HOME/.m2/settings.xml
+- export MAVEN_OPTS="-Dmaven.artifact.threads=3 -Xmx2469m -XX:MaxPermSize=512m"
+- export PROJECTS="--projects !extensions/voice/org.eclipse.smarthome.voice.mactts/pom.xml"
```

Comment:
No, as Travis runs on Linux...

On Wed, Jun 8, 2016 at 11:36 AM, Kelly Davis <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/mozilla/smarthome/pull/65#discussion_r66223932>:
>
> > @@ -0,0 +1,42 @@
> > +language: java
> > +jdk: openjdk8
> > +sudo: required
> > +dist: trusty
> > +
> > +install:
> > +- wget http://www-us.apache.org/dist/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
> > +- tar xzf apache-maven-3.3.9-bin.tar.gz
> > +- curl -s -L https://github.com/mozilla/smarthome/releases/latest | egrep -o /mozilla/smarthome/releases/download/.*/m2repository.tar.gz | wget --base=http://github.com/ -i -
> > +- tar xf m2repository.tar.gz -C $HOME/.m2/repository/
> > +
> > +before_script:
> > +- rm $HOME/.m2/settings.xml
> > +- export MAVEN_OPTS="-Dmaven.artifact.threads=3 -Xmx2469m -XX:MaxPermSize=512m"
> > +- export PROJECTS="--projects !extensions/voice/org.eclipse.smarthome.voice.mactts/pom.xml"
>
> Haven't looked at the entire pull request, it's big, but if you've moved
> the mactts, shouldn't this .travis.yml
> not have to exclude mactts as mactts should compile and run.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mozilla/smarthome/pull/65/files/4c95be33fdfa2d55dd6650be6bea66d1f7c39b05#r66223932>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFtqsPdyYeNFp4ZyrUmSTDLwKjfkfk3Lks5qJo0WgaJpZM4IwrNd>
> .
>